openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -68,6 +68,8 @@ class AccountView(APIView):
 
                 * goals: null or textual representation of goals
 
+                * bio: null or textural representation of user biographical information ("about me")
```

Comment:
No HTML allowed in bio. It might be nice to enforce a size limit so people don't type manifestos, but not required.